R/processing.R

```diff
@@ -29,18 +29,12 @@ add_extra_na_cols <- function(data) {
 #' the datasets should always be > 0.
 #' @param data A data frame
 #' @return A data frame with all relevant data > 0.
+#' @importFrom dplyr mutate_if
 #' @concept utility
 set_negative_values_to_zero <- function(data) {
-  numeric_col_names <- c(
-    "deaths_total", "cases_total", "recovered_total", "hosp_total",
-    "tested_total", "cases_new", "deaths_new", "recovered_new", "hosp_new",
-    "tested_new"
+  data <- suppressMessages(
+    mutate_if(data, is.numeric, ~ replace(., . < 0, 0))
```

Comment:
does `.` here generate a CRAN check note?

Reply:
looks good to me